content/applications/hr/fleet/configuration.rst

```diff
@@ -0,0 +1,234 @@
++-------+--------------+-----------+---------------+
+|       |              | Class SLS |               |
++-------+--------------+-----------+---------------+
+
+.. _configuration/add-model:
+
+Add a new model
+---------------
+
+New vehicle models can easily be added to the database. To add a new model, navigate to
+:menuselection:`Fleet app --> Configuration --> Vehicle Models`. Click :guilabel:`Create` and a
+vehicle model form will load. Enter the following information on the form, then click
+:guilabel:`Save`. Be advised, some fields are specific to Belgian based companies, so not all fields
+or sections may be visible depending on the location of the company.
+
+- :guilabel:`Model name`: Enter the model name in the field.
+- :guilabel:`Manufacturer`: Select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  & Edit`.
+- :guilabel:`Vehicle Type`: Select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application. Adding additional vehicle types is not possible as it
+  will affect payroll.
+- :guilabel:`Category`: Select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create
+  (new category)`.
+
+.. note::
+   When the manufacturer is selected, the logo for the manufacturer will automatically load in the
+   image box in the top right corner.
+
+Information tab
+~~~~~~~~~~~~~~~
+
+Model
+*****
+
+- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
+- :guilabel:`Color`: Enter the color of the vehicle.
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
+
+Salary
+******
+
+- :guilabel:`Can be requested`: Check this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: This value is specific to Belgian based companies only. If logged into a
+  Belgian company, this value is automatically calculated based on Belgian laws and regulations, and
+  cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions` field
+  in the :guilabel:`Engine` section of the vehicle form. Modifying the :guilabel:`CO2 Emissions`
+  field will adjust the value in the :guilabel:`CO2 fee` field.
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
```

Comment:
again salary configuration is familiar to you, but I have no idea where this comes from